### DISCUSSION TITLE
UI Rework

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#93] [General]
![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2012-37-00.png)

**Comment by 0xMRTT** [comment]
What's the name of icon in the view switcher ?

**Reply by daudix** [reply]
It's an icon from [Icon Library](https://flathub.org/apps/details/org.gnome.design.IconLibrary), it need to specified with Gresource

**Reply by daudix** [reply]
Brush is "brush", Monet is "pipette", CSS is "code"

**Comment by ArtyIF** [comment]
I'd say make the CSS tab a "plugin settings" tab, since not every planned platform supports CSS.15:25, August 10, 2022, 0xMRTT ***@***.***>:

**Reply by daudix** [reply]
Will make it in V3

**Reply by 0xMRTT** [reply]
![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2014-43-37.png)

**Comment by 0xMRTT** [comment]
What's the name of icons for unsaved change

**Reply by daudix** [reply]
This is custom icons, it need to be specified using same Gresource

**Reply by 0xMRTT** [reply]
Yes, but where are icons ?. In Icon Library ?

**Reply by 0xMRTT** [reply]
Found, in design repo

**Reply by daudix** [reply]
[CustomIcons.zip](https://github.com/AdwCustomizerTeam/AdwCustomizer/files/9300193/CustomIcons.zip)

**Comment by 0xMRTT** [comment]
## No changes

![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2015-06-53.png)

## Unsaved change

![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2015-07-02.png)

**Comment by 0xMRTT** [comment]
So, i keep the CSS in `colors` tab ?

**Reply by daudix** [reply]
I think it can be placed tab 4

**Reply by ArtyIF** [reply]
No, CSS would be in Plugins tab. It's mostly related only to GTK 4 and GTK 3.

**Comment by daudix** [comment]
Here are V3, plugins tab are basically UI of [Extension Manager](https://beta.flathub.org/ru/apps/details/com.mattjakeman.ExtensionManager)

# V3

![Adwaita Manager V3](https://user-images.githubusercontent.com/77155297/183916859-f725aa23-04d3-4578-91b3-c767a041a138.png)

**Reply by 0xMRTT** [reply]
Could you add this mockup in Design repo. Please

**Reply by daudix** [reply]
Working on it

**Reply by daudix** [reply]
Done

**Comment by 0xMRTT** [comment]
Unsaved and saved icons doesn't work with dark theme

![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2016-19-12.png)

**Reply by daudix** [reply]
Will try to fix this

**Comment by 0xMRTT** [comment]
![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2016-52-38.png)

added monet page

I need to modify https://github.com/avanishsubbiah/material-color-utilities-python for integrating with flatpak and meson build system.

**Reply by daudix** [reply]
Just a question, can app automaticly pull the wallpaper?

**Reply by 0xMRTT** [reply]
i'm working on

**Comment by daudix** [comment]
Fixed disk icons, took me a bit more than I expected, but at least I will know how to do this in the future :/

[📎 disk.zip](https://github.com/AdwCustomizerTeam/AdwCustomizer/files/9301326/disk.zip)

**Comment by 0xMRTT** [comment]
Added toast

**Comment by 0xMRTT** [comment]
Should i color the button in yellow when there is an unsaved change because it's hard to see when there is an unsaved change

**Reply by daudix** [reply]
not sure, but if you want, here is modified icon for it

[📎 disk-unsaved-fill-symbolic.zip](https://github.com/AdwCustomizerTeam/AdwCustomizer/files/9301397/disk-unsaved-fill-symbolic.zip)

**Reply by daudix** [reply]
added fully coloured icon, it colors only when it in app

**Reply by daudix** [reply]
But even on my low-res display, from your screenshot, it visible pretty good

**Reply by 0xMRTT** [reply]
I'll change nothing

**Reply by 0xMRTT** [reply]
But it will be great if when you have unsaved change and you want to close the app, the app ask you to save

**Reply by daudix** [reply]
Yes!

**Comment by daudix** [comment]
Maybe rename plugins tab to "Advanced" ? as it have not only plugins, but also CSS

**Reply by 0xMRTT** [reply]
done

**Comment by 0xMRTT** [comment]
Hi @daudix-UFO 

I found the way to implement the welcome screen. But on the first page, there is an image. Could you add this image in the design repo ?

**Reply by daudix** [reply]
Thank you for reminder, I really forgot about it :/

**Reply by daudix** [reply]
Now it aviable in the repo! (totally new, it may change)

**Comment by 0xMRTT** [comment]
On the mockups, next to the apply button there is a download icon. What's the usage of this button ?

**Reply by daudix** [reply]
Well this was used to save preset, but now it replaced with disc icon, will remove it soon
